### DISCUSSION TITLE
Don't lock discussions when "already tracked in proposal"

### DISCUSSION THREAD

**Discussion by xix-xeaon** [godotengine/godot-proposals#4655] [General Discussions]
These discussions are especially useful for gauging user interest in certain features. The "upvote" system here is superior to the one connected with issues, and these discussions are more open and accessible to users which makes the count more representative.

For instance, the post https://github.com/godotengine/godot-proposals/discussions/4641 already had 3 upvotes after only a few hours, while the linked proposal only has 4 after more than a year. I was however unable to upvote it because it had already been locked. It's also nice to keep track of features I'm interested in, although I don't believe there is yet a way to filter based on your personal upvotes.

Please let users discuss features and upvote discussions freely and without unnecessary moderation here - leave strict moderation to the proposals where it belongs.

**Comment by Zireael07** [comment]
I wouldn't say upvotes here is superior to leaving a +1 on an issue.

You can't filter either discussions or issues based on your upvotes/reactions, so that's a moot point.

However, there is an external site for tracking proposal issues.

The team is already swamped with issues/proposals, so I can see why they want to keep discussion of one feature in one place. Especially as the mod team for proposals seems to be limited to two people?

**Comment by Calinou** [comment]
> These discussions are especially useful for gauging user interest in certain features. The "upvote" system here is superior to the one connected with issues, and these discussions are more open and accessible to users which makes the count more representative.

Votes on discussions are not transferred to the actual proposal issues, which makes [sorting by +1 reactions](https://github.com/godotengine/godot-proposals/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) less meaningful. This also applies to the [Godot proposals viewer](godot-proposals-viewer.github.io/).

In general, having duplicate discussion threads and proposal issues is not very useful for contributors to base decisions on. It makes things harder to follow and can result in insightful comments (or even implementation work) being entirely missed by people reading only one of the threads. I've seen this happen time and time again over the years :slightly_smiling_face: 

Also, remember that you can leave a +1 reaction on https://github.com/godotengine/godot-proposals/issues/2361 too. While you cannot add reactions on locked issues/discussions, you can add reactions on linked issues/discussions within those locked threads.

> although I don't believe there is yet a way to filter based on your personal upvotes.

Neither issues nor discussions have this feature, so I would recommend using browser bookmarks, a private user project, or even a private [gist](https://gist.github.com).

![image](https://user-images.githubusercontent.com/180032/173465711-a983d8e7-e029-4aad-82f4-598524dbc524.png)
